Python/Object_detection/readme.md

```diff
@@ -0,0 +1,31 @@
+# Object_Detection
+
+It solves the issue #982 .
+Detects the object using python.
+
+
+## Technology Stack Used
+
+
+![Python](https://img.shields.io/badge/Python-yellow.svg?logo=PHP&style=flat-square)
```

Comment:
Rather than Tech Stacks, mention the dependencies, like the modules or any third-party API used.

Do one thing, please fix the README using this [template](https://github.com/HarshCasper/Rotten-Scripts/blob/master/TEMPLATE_README.md)